src/js/cards-rendering.js

```diff
@@ -0,0 +1,163 @@
+import { emptyStar, fullStar, halfStar } from './stars';
```

Comment:
a czemu tu jest card rendering jak one się renderują w innym miejscu już i tam się tworzą, to jest jedynie do działania w my-library?
no i gwiazdku to Daniel z nimi walczył z nimi i już się super wyświetlały te jego, czemu chcesz wstawić inne?
generalnie chyba to wszystko trafia w endpoint na naszej stronie my-library więc ja to puszczam, ale to jest do skoordynowania i ograniczenia się do jednego z rozwiązań, żebyśmy nie mnożyli tych samych spraw